src/dataset-builder/DemographicsChart.ts

```diff
@@ -7,6 +7,7 @@ interface ChartLabel {
 }
 export interface ChartSeries {
   name?: string;
+  colorByPoint?: boolean;
```

Comment:
what's this?

Reply:
It makes the bars different colors?

Reply:
Yes, exactly! More info here: https://api.highcharts.com/highcharts/series.column.colorByPoint